recipe/build.py

```diff
@@ -103,6 +103,8 @@ def __init__(self, plat, version, version_patch, runfile):
             self.cuda_libraries.append("cublasLt")
         if self.major_minor >= (10, 2):
             self.cuda_libraries.append("nvjpeg")
+        if self.major_minor < (11, 0):
+            self.cuda_libraries.append("nppicom")
```

Comment:
FWIW it looks like this file is still present on Windows CUDA 11.0. So we might want to adjust this accordingly. I don't have Windows so can't confirm, but it is noted in [Attachment A]( https://docs.nvidia.com/cuda/archive/11.0/eula/index.html#attachment-a ).

Reply:
I can't see these on my current CUDA 11 Windows installation.

Reply:
Ok maybe the attachment is listing previously covered libraries then.

Reply:
The attachment for CUDA 10.0 was inaccurate with respect to libnvjpeg on Windows, so I'd say they are not up-to-date.